log.graphqls

```diff
@@ -75,8 +75,33 @@ enum ContentType {
     YAML
 }
 
+input LogTestRequest {
+    # The log data of protocol https://github.com/apache/skywalking-data-collect-protocol/blob/e626ee04850703c220f64b642d2893fa65572943/logging/Logging.proto#41
+    # in JSON format
+    log: String!
+    dsl: String!
+}
+
+type LogTestResponse {
+    # The final log if it will be persisted, this can be empty if the log is dropped.
+    log: Log
+    # The metrics generated during the LAL when testing a LogTestRequest
+    metrics: [LogTestMetrics!]
+}
+
+# The metrics generated during the LAL when testing a LogTestRequest
+type LogTestMetrics {
+    name: String!
+    tags: [KeyValue!]
+    value: Long!
+    timestamp: Long!
+}
+
 extend type Query {
     # Return true if the current storage implementation supports fuzzy query for logs.
     supportQueryLogsByKeywords: Boolean!
     queryLogs(condition: LogQueryCondition): Logs
+
+    # Test the logs and get the results of the LAL output.
+    test(requests: [LogTestRequest!]): [LogTestResponse!]
```

Comment:
I think this response has a little issue. The metrics are generated through the list<LogTestRequest>, right? Then, `metrics` would be grouped by different LogTestResponse. I think we should have only one LogTestResponse with `[Log!]` in it, right?

Reply:
The idea is that every single `LogTestRequest` will get a `LogTestResponse`, and `LogTestResponse` contains the `log` and the `metrics` list it generates

Reply:
But I think it shouldn't get 1:1 response. If we do that, there is less point to accept a list from backend perspective.

Reply:
Because, 1:1 could be assembled as a list by UI or CLI itself.

Reply:
Then what we can do is to merge the `[LogTestResponse!]` into one, but we cannot merge the metrics generated from all logs, it involves with MAL and streaming process

Reply:
OK, if it can't be merged(due to streaming downsampling not involved in LAL), let's keep one to one. 
@Fine0830 could still write UI in multiple logs, if they can be visualized well, by using GraphQL to assemble query.